feature/contributor/src/main/java/com/droidknights/app/feature/contributor/model/convert/ContributorsConvert.kt

```diff
@@ -0,0 +1,32 @@
+package com.droidknights.app.feature.contributor.model.convert
+
+import com.droidknights.app.core.model.Contributor
+import com.droidknights.app.feature.contributor.model.ContributorsUiState
+import kotlinx.collections.immutable.toPersistentList
+
+internal fun Map<Int, List<Contributor>>.toContributorsUiState(): ContributorsUiState {
+    val newList = mutableListOf<ContributorsUiState.Contributors.Item>()
+
+    // key, value 추가
+    forEach { (key, values) ->
+        newList.add(
+            ContributorsUiState.Contributors.Item.Section(
+                title = key.toString(),
+            )
+        )
+        values.forEach { item ->
+            newList.add(
+                ContributorsUiState.Contributors.Item.User(
+                    id = item.id,
+                    imageUrl = item.imageUrl,
+                    githubUrl = item.githubUrl,
+                    name = item.name,
+                )
+            )
+        }
+    }
```

Comment:
flatMap을 활용해보면 어떨까요?
```suggestion
    val newList = this.flatMap { (key, values) ->
        sequenceOf(
            ContributorsUiState.Contributors.Item.Section(title = key.toString())
        ) + values.map { item ->
            ContributorsUiState.Contributors.Item.User(
                id = item.id,
                imageUrl = item.imageUrl,
                githubUrl = item.githubUrl,
                name = item.name
            )
        }
    }
```

Reply:
오 어떻게 할까 고민하다가 한건데 좋은 방법이네요.